rabbitmq/json-exchanges.yml

```diff
@@ -0,0 +1,123 @@
+type: json
+data:
+  url: /api/exchanges
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: exchanges
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name} && @.vhost=${vhost})]
+
+    metric:
+      - name: exchanges.message_stats.ack
+        source: message_stats/ack
+        type: long_gauge
+        label: ack messages
+        description: Number of messages delivered to clients and acknowledged
+
+      - name: exchanges.message_stats.ack.rate
```

Comment:
Rate as reported by monitored service is typically useless, it has no info about time frame, so rate metric (if we want to display it) should be calculated by UI/reports.

Reply:
I will remove all rate metrics.